core/language/en-GB/Fields.multids

```diff
@@ -1,5 +1,8 @@
 title: $:/language/Docs/Fields/
 
+table-header-name-20240804162913000: Name
```

Comment:
What is the thinking behind including a timestamp in the fieldname?

Reply:
I wanted to have a unique field-name to avoid name clashes. `table-header-name` is a perfectly fine field-name that users could use. I thought it is very unlikely, that someone will use an "outdated" date for their field-names in the future. Firs I did use `-xyzabc`. It's unlikely to cause name clashes, but I was more confident with the date.

I'm open to change the name